parametric/apps/java/src/main/proto/apm_test_client.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
```

Comment:
Is there a way we can reuse https://github.com/DataDog/system-tests/blob/main/parametric/protos/apm_test_client.proto?

Reply:
I think I can configure the maven plugin to use the original file. 
I made a copy because .net client did it too.